docs/src/lessons/XYPlots/XYPlotsLesson.js

```diff
@@ -8,8 +8,8 @@ export default class XYPlotsLesson extends React.Component {
     return <Lesson name="XY Plots" componentName="XYPlots">
       <p>
         Nearly all of the chart components included in Reactochart (except PieChart and TreeMap)
-        are cartesian or "X/Y"-type charts. They accept an array of <code>data</code>
-        and represent these data with some form of SVG marks on a 2-dimensional X/Y coordinate plane.
+        are cartesian or "X/Y"-type charts. They accept an array of <code>data</code> and
```

Comment:
This wasn't actually a typo, some of the `<code>` blocks seem to collide with the following word if there's a line break in between them.